Sources/Remote Logging/Crash Logging/CrashLogging.swift

```diff
@@ -178,14 +178,18 @@ public extension CrashLogging {
             return
         }
 
+        try submitSerializedEvents(payload: requestBody, callback: callback)
```

Comment:
What does this do?

Reply:
I've extracted this method from [logErrorsImmediately](https://github.com/Automattic/Automattic-Tracks-iOS/blob/develop/Sources/Remote%20Logging/Crash%20Logging/CrashLogging.swift#L158) (develop branch).

Essentially, it's hitting the Sentry endpoint, **right away**, via HTTP.

This is done this way, so that the crashlog is not lost (in the low disk space eventuality). Plus this allows us to submit the crashlog synchronously.

OR, in the `Log Panic` mechanism scenario, we get to report Sentry Events, synchronously.

Reply:
Mmmmm, wait so where is `submitSerializedEvents(payload: requestBody, callback: callback)` defined?

Reply:
Ah, okay, I see what's happening, thanks.